api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImplTest.java

```diff
@@ -246,6 +246,7 @@ public void shouldReturnCollectionOfPeopleSortedByName() {
 		
 		// Smallest given name of person i should be less than the largest given name of person i + 1.
 		for (int i = 1; i < people.size(); i++) {
+			assertThat(people.get(i-1).getNames(), not(empty())); // Not sure why this test is failing
```

Comment:
@ibacher I'm not sure why this test is failing. 

Also, your param to prop changes are in this PR?